addons/website/static/src/builder/plugins/customize_website_plugin.js

```diff
@@ -168,8 +167,10 @@ export class CustomizeWebsitePlugin extends Plugin {
                     if (!save) {
                         return;
                     }
+                    document
+                        .querySelector(".o-website-builder_sidebar")
+                        .classList.remove("o_builder_sidebar_open");
```

Comment:
maybe add a clean way to do it with state ? Builder should maybe give access to function for display/hide builder ?